.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+## What does this PR do?
+*(A few words about the PR)*
```

Comment:
I can't find any specific info, but I'm almost sure that you should add new lines. Check https://www.markdownguide.org/getting-started/ as a reference.

Reply:
Sounds reasonable, I will change that 👍